examples/flutter_gallery/ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This seems like an odd file to appear in this PR?

Reply:
A bit of a drive by. But the git state is always dirty in the gallery since #21994